aspnetcore/signalr/redis-backplane.md

```diff
@@ -56,50 +55,49 @@ This article explains SignalR-specific aspects of setting up a [Redis](https://r
 
 ::: moniker-end
 
-::: moniker range="> aspnetcore-2.1"
+::: moniker range="= aspnetcore-2.2"
 
 * In the SignalR app, install one of the following NuGet packages:
 
   * `Microsoft.AspNetCore.SignalR.StackExchangeRedis` - Depends on StackExchange.Redis 2.X.X. This is the recommended package for ASP.NET Core 2.2 and later.
-  * `Microsoft.AspNetCore.SignalR.Redis` - Depends on StackExchange.Redis 1.X.X. This package will not be shipping in ASP.NET Core 3.0.
-
-* In the `Startup.ConfigureServices` method, call `AddStackExchangeRedis` after `AddSignalR`:
+  * `Microsoft.AspNetCore.SignalR.Redis` - Depends on StackExchange.Redis 1.X.X. This package isn't included in ASP.NET Core 3.0 and later.
 
-Note: If using `Microsoft.AspNetCore.SignalR.Redis` then call `AddRedis`
+* In the `Startup.ConfigureServices` method, call <xref:Microsoft.Extensions.DependencyInjection.StackExchangeRedisDependencyInjectionExtensions.AddStackExchangeRedis*>:
 
   ```csharp
   services.AddSignalR().AddStackExchangeRedis("<your_Redis_connection_string>");
   ```
 
+ When using `Microsoft.AspNetCore.SignalR.Redis`, call <xref:Microsoft.Extensions.DependencyInjection.RedisDependencyInjectionExtensions.AddRedis*>.
```

Comment:
neat 👍.  I assume that create some type of hyperlink? 🤔

Reply:
Yes